src/fu-util-common.c

```diff
@@ -1374,6 +1374,10 @@ fu_util_device_problem_to_string(FwupdClient *client, FwupdDevice *dev, FwupdDev
 		/* TRANSLATORS: emulated means we are pretending to be a different model */
 		return g_strdup(_("Device is emulated"));
 	}
+	if (problem == FWUPD_DEVICE_PROBLEM_MISSING_LICENSE) {
+		/* TRANSLATORS: the update function is an add-on feature sold separately" */
+		return g_strdup(_("Device requires a software license to update"));
```

Comment:
This string is different than the other problem string in the PR.  Shouldn't they be the same?